src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingCustomShardAllocationSpec.cs

```diff
@@ -28,29 +28,29 @@ public ClusterShardingCustomShardAllocationSpecConfig()
             Second = Role("second");
 
             CommonConfig = DebugConfig(false)
-                .WithFallback(ConfigurationFactory.ParseString(@"
-                    akka.actor {
-                        serializers {
+                .WithFallback(ConfigurationFactory.ParseString($@"
+                    akka.actor {{
+                        serializers {{
                             hyperion = ""Akka.Serialization.HyperionSerializer, Akka.Serialization.Hyperion""
-                        }
-                        serialization-bindings {
+                        }}
+                        serialization-bindings {{
                             ""System.Object"" = hyperion
-                        }
-                    }
-
+                        }}
+                    }}
+                    akka.loglevel = {LogLevel}
```

Comment:
So, the idea is that, let's try to not modify any specs - let them write the logs as there were logging before. Let's just make MultiNodeTestRunner to not output logs of particular level to the console / test output provider.

Reply:
I will not leave a comment under each modification like this, but you got the idea

Reply:
> 
> 
> So, the idea is that, let's try to not modify any specs - let them write the logs as there were logging before. Let's just make MultiNodeTestRunner to not output logs of particular level to the console / test output provider.

I think we are not really modifying any Specs - `MultiNodeConfig` is what is being modified.

Reply:
I mean, we should try to not modify the specs configuration - just the way how MNTR process is printing this logs to the output.